Source/Urho3D/Scene/Serializable.h

```diff
@@ -225,6 +225,11 @@ SharedPtr<AttributeAccessor> MakeVariantAttributeAccessor(TGetFunction getFuncti
     [](const ClassName& self, Urho3D::Variant& value) { value = static_cast<int>(self.getFunction()); }, \
     [](ClassName& self, const Urho3D::Variant& value) { self.setFunction(static_cast<typeName>(value.Get<int>())); })
 
+/// Make get/set object attribute accessor.
+#define URHO3D_MAKE_GET_SET_OBJECT_ATTRIBUTE_ACCESSOR(variable) Urho3D::MakeVariantAttributeAccessor<ClassName>( \
```

Comment:
`URHO3D_MAKE_MEMBER_ATTRIBUTE_ACCESSOR` is more fitting name, if you look ath other macros. Accessor is about get/set functions.

Reply:
Are you sure about this? Both `bool foo_;` and `SharedPtr<Node> foo_;` are members, but only `SharedPtr<Node> foo_;` is an object. How about `URHO3D_MAKE_OBJECT_ATTRIBUTE_ACCESSOR`?